crates/hyperdrive-math/src/lp.rs

```diff
@@ -7,6 +7,21 @@ use fixed_point_macros::{fixed, int256};
 use crate::{State, YieldSpace};
 
 impl State {
+    /// Calculates the number of base that are not reserved by open positions.
+    pub fn calculate_idle_share_reserves_in_base(&self) -> FixedPoint {
+        // NOTE: Round up to underestimate the pool's idle.
+        let long_exposure = self.long_exposure().div_up(self.vault_share_price());
+
+        let mut idle_shares_in_base = fixed!(0e18);
```

Comment:
Nit: Can we add a comment here? Something like this:

```suggestion
        // Calculate the idle base reserves.
        let mut idle_shares_in_base = fixed!(0e18);
```